src/sema/statements.rs

```diff
@@ -293,6 +293,7 @@ pub fn resolve_function_body(
 }
 
 /// Resolve a statement
+#[allow(clippy::ptr_arg)]
```

Comment:
What did add this?

Reply:
The commit description could be better. 

I've been testing rust beta channel, so that things work when rust 1.59.0 is released later today. This release includes a bizarre clippy warning that says that `& mut Vec<_>` parameter can be replaced with `&mut [_]` which is not the case because we want to push onto the vector.

So we silence the clippy warning.

Reply:
I've improved the commit description